samtranslator/model/sam_resources.py

```diff
@@ -2422,6 +2429,26 @@ def _construct_appsync_schema(
 
         return schema
 
+    def _construct_appsync_api_keys(
+        self, api_keys: Dict[str, aws_serverless_graphqlapi.ApiKey], api_id: Intrinsicable[str]
+    ) -> List[Resource]:
+        resources: List[Resource] = []
+
+        # TODO: Add datetime parsing for ExpiresOn
```

Comment:
```suggestion
        # TODO: Add datetime parsing for ExpiresOn; currently expects Unix timestamp
```

Reply:
That's so nice that I can just add requested changes like that